sdk/src/types/block/mana/rewards.rs

```diff
@@ -0,0 +1,49 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use getset::CopyGetters;
+use packable::Packable;
+
+use crate::types::block::{slot::EpochIndex, Error};
+
+#[derive(Copy, Clone, Debug, Eq, PartialEq, Ord, PartialOrd, Packable, CopyGetters)]
+#[cfg_attr(
+    feature = "serde",
+    derive(serde::Serialize, serde::Deserialize),
+    serde(rename_all = "camelCase")
+)]
+#[packable(unpack_error = Error)]
+#[getset(get_copy = "pub")]
+pub struct RewardsParameters {
+    /// The number of validation blocks that should be issued by a selected validator per slot during its epoch duties.
+    validator_blocks_per_slot: u8,
+    /// Used for shift operation for calculation of profit margin.
+    profit_margin_exponent: u8,
+    /// The length in epochs of the bootstrapping phase.
+    bootstrapping_duration: EpochIndex,
+    /// The coefficient used for calculation of initial rewards.
+    #[cfg_attr(feature = "serde", serde(with = "crate::utils::serde::string"))]
+    mana_share_coefficient: u64,
+    /// The exponent used for calculation of the initial reward.
+    decay_balancing_constant_exponent: u8,
+    /// Needs to be an integer approximation calculated based on chosen DecayBalancingConstantExponent.
```

Comment:
```suggestion
    /// An integer approximation which is calculated using the `decay_balancing_constant_exponent`.
```
Though an overall better description is needed I think